charts/plane-enterprise/templates/workloads/pi-api.deployment.yaml

```
@@ -0,0 +1,92 @@
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+      serviceAccount: {{ .Release.Name }}-srv-account
+      serviceAccountName: {{ .Release.Name }}-srv-account
+
+---
+{{- end }}
+
+
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (4 > 0)

(empty-lines)